Source/Flow/Private/FlowAsset.cpp

```diff
@@ -189,35 +189,73 @@ void UFlowAsset::HarvestNodeConnections()
 	}
 }
 
-void UFlowAsset::AddCustomInput(const FName& InName)
+void UFlowAsset::AddCustomInput(const FName& EventName)
 {
-	if (!CustomInputs.Contains(InName))
+	if (!CustomInputs.Contains(EventName))
 	{
-		CustomInputs.Add(InName);
+		CustomInputs.Add(EventName);
 	}
 }
 
-void UFlowAsset::RemoveCustomInput(const FName& InName)
+void UFlowAsset::RemoveCustomInput(const FName& EventName)
 {
-	if (CustomInputs.Contains(InName))
+	if (CustomInputs.Contains(EventName))
 	{
-		CustomInputs.Remove(InName);
+		CustomInputs.Remove(EventName);
 	}
 }
+
+void UFlowAsset::AddCustomOutput(const FName& EventName)
+{
+	check(!CustomOutputs.Contains(EventName));
+
+	CustomOutputs.Add(EventName);
+}
+
+void UFlowAsset::RemoveCustomOutput(const FName& EventName)
+{
+	check(CustomOutputs.Contains(EventName));
```

Comment:
Any code calling these functions should not be double-adding or double removing custom events, so I'm asserting it as a code error.

Reply:
I understand motivation, but I'd like to keep consistent with the rest of the plugin's code so... I converted checks to regular If statements.  Might review this approach later ;)